src/ambassadors/core.ts

```diff
@@ -793,6 +909,10 @@ const ambassadors = {
       "Chips Ahoy was bred as a feeder rat for snakes, but she will now live out her full life at Alveus.",
     mission:
       "She is an ambassadors for how rodenticide use and outdoor cats affect all levels of wildlife.",
+    native: {
```

Comment:
I would probably say that these rats (`f. domestica`) are worldwide are they are domesticated?

Reply:
https://en.wikipedia.org/wiki/Fancy_rat might also be a better wiki article for the domesticated rat?

Reply:
ah gotcha, my mistake. thank you for catching that